packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -297,12 +304,18 @@ export class VerticalStackedBarChartBase extends React.Component<
           ? xScale(lineObject[item][i - 1].xItem.xAxisPoint as number)
           : // eslint-disable-next-line @typescript-eslint/no-explicit-any
             (xBarScale as any)(lineObject[item][i - 1].xItem.xAxisPoint as string);
-        const y1 = yScale(lineObject[item][i - 1].y);
+        const y1 =
+          lineObject[item][i - 1].isUsingSecondaryYScale && secondaryYScale
```

Comment:
both y1 and y2 should be on secondary scale to be able to draw a meaningful line